students/BrianNisonger/Lesson2/assignment/charges_calc.py

```diff
@@ -0,0 +1,125 @@
+#pylint: disable=W0703
+'''
+Returns total price paid for individual rentals
+'''
+import argparse
+import json
+import datetime
+import math
+import logging
+
+
+def parse_cmd_arguments():
+    """
+    Parses commands from the commandline
+    """
+    parser = argparse.ArgumentParser(description='Process some integers.')
+    parser.add_argument('-i', '--input', help='input JSON file', required=True)
+    parser.add_argument(
+        '-o', '--output', help='ouput JSON file', required=True)
+    parser.add_argument(
+        '-d', '--debug', help='debugger level', required=False, default=0)
+    return parser.parse_args()
+
+
+def configure_logging(debug_level):
+    """
+    Configures the logging
+    """
+    format_logger = '%(asctime)s %(filename)s:%(lineno)-3d%(levelname)s'
+    format_logger += ' %(message)s'
+    log_file = datetime.datetime.now().strftime('%Y-%m-%d') + '.log'
+    debug_level = int(debug_level)
+    if debug_level == 0:
+        logging.basicConfig(level=logging.CRITICAL)
```

Comment:
If you're not going to log, no point in all of the logging overhead. Note that if someone did stick in a critical log message it would show up in the console.